production/ksonnet/loki/config.libsonnet

```diff
@@ -9,18 +9,30 @@
 
     grpc_server_max_msg_size: 100 << 20,  // 100MB
 
+    annotations: {},
```

Comment:
If we are doing annotations, labels as well ?

Reply:
done